packet/packet.go

```diff
@@ -154,6 +154,14 @@ func (hdr *UDPHdr) String() string {
 	return r0 + r1 + r2
 }
 
+type ICMPHdr struct {
+	Type       uint8  // ICMP message type
+	Code       uint8  // ICMP message code
+	Cksum      uint16 // ICMP checksum
+	Identifier uint16 // ICMP message identifier in some messages
+	SeqNum     uint16 // ICMP message sequence number in some messages
+}
+
```

Comment:
Can you please add at the beginning of packet.go file comment, that we not support set of parsing functions for ICMP.